src/TypeCatalogParser/Main.cs

```diff
@@ -34,7 +34,7 @@ public static void Main(string[] args)
                                          // Get the real reference assemblies
                                          from y in x.CompileTimeAssemblies where y.Path.EndsWith(".dll")
                                          // Construct the path to the assemblies
-                                         select $"{context.PackagesDirectory}/{x.Name}/{x.Version}/{y.Path};");
+                                         select $"{context.PackagesDirectory}/{x.Name.ToLower()}/{x.Version}/{y.Path};");
```

Comment:
Is it expected/desired for this to be dependent on the current culture, or should this be `ToLowerInvariant`?